src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class Contact {
+
+    public final String value;
+    private boolean isPrivate;
+  
+    /**
+     * Validates given contact.
```

Comment:
the constructor is not doing any validation here; you are misleading the users who write client code!